src/applications/form-renderer/utils/formConfig.js

```diff
@@ -4,29 +4,51 @@ import {
   dateOfBirthUI,
   fullNameSchema,
   fullNameUI,
+  serviceNumberSchema,
+  serviceNumberUI,
+  ssnOrVaFileNumberSchema,
+  ssnOrVaFileNumberUI,
   titleUI,
 } from 'platform/forms-system/src/js/web-component-patterns';
```

Comment:
Do we need to worry about how absolutely monstrous this import will become as we add more schemas to the renderer application?

Reply:
Short answer, yes. This has been on my mind in a number of places (here, KISS file, etc.). My approach has largely been to build now and optimize later. I do think this will become noteworthy once we have a collection of components/patterns/schemas for "all" forms. For now, though, this should be no different than any other form, as we only currently have components/patterns/schemas for a single form.

I think the "solution" likely includes dynamic loading of bundles on a per-page basis. I'm actually not sure how much of this falls to us vs. to the Forms Library. I'm also not sure how much of this might be in place already 🤷. As I said, though, I don't want to worry about premature optimization. It will be enough of a lift just to build this thing. We need to get it working and then figure out if we need to optimize.